Include/internal/pycore_runtime.h

```diff
@@ -53,12 +53,56 @@ typedef struct _Py_AuditHookEntry {
     void *userData;
 } _Py_AuditHookEntry;
 
+typedef struct _Py_DebugOffsets {
```

Comment:
Would it make sense to break this up into smaller structs and have the main struct point to those?
That way additional fields can be added to the thread state data without disturbing the frame data.

Reply:
Good idea, you want those smaller structs inlined or as separate (named) structs?

Reply:
Either works

Reply:
Done